### DISCUSSION TITLE
how to extend timeout for readiness probe

### DISCUSSION THREAD

**Discussion by m2khosravi** [strimzi/strimzi-kafka-operator#10033] [Q&A]
Hi Team, I can see that the readiness probe timeout configuration has been set to 5 seconds by default for Kafka pods (ProbeUtils.java)

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/47779657/5c7cd31b-b2d9-4b05-8bd1-aeaaea0ac1f8)

and defined here for Kafka (KafkaCluster.java)

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/47779657/d32c99c1-5a31-4f1e-80ff-fd509e8cae34)

How can I modify this timeout setting to increase it?

**Comment by scholzj** [comment]
Did you checked the documentation? It for sure covers configuring the
probes.

**Comment by m2khosravi** [comment]
Thanks for your reply.

from the [documentation](https://strimzi.io/docs/operators/in-development/deploying#con-config-kafka-str), I can see it's possible to set this configuration when deploying a cluster 

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/47779657/0592522b-6d6d-465a-b52f-da33b604d6e0)

but is there any way to make it cluster-wide and change it by default without setting it for each specific deployment?

**Comment by scholzj** [comment]
No, you configure this in the custom resources.
